### PR TITLE
Update 0004-AMND-amends.md

### PR DESCRIPTION
**Pull request by vega-paul**
Requirement to  cover/exercise the order amend panic case which occurred on testnet

### CODE REVIEW

File: protocol/0004-AMND-amends.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 - Attempting to alter details on a filled order will cause the amend to be rejected (<a name="0004-AMND-017" href="#0004-AMND-017">0004-AMND-017</a>)
 - Attempting to alter details on a cancelled order will cause the amend to be rejected (<a name="0004-AMND-018" href="#0004-AMND-018">0004-AMND-018</a>)
 - Attempting to alter details on an expired order will cause the amend to be rejected (<a name="0004-AMND-019" href="#0004-AMND-019">0004-AMND-019</a>)
+- Amending expiry time of an active GTT order to a past time whilst also simultaneously amending the price and size of the GTT order will insert the order at the back of the queue at the new price level with the new size and expiry time causing the order to immediately expire (<a name="0004-AMND-029" href="#0004-AMND-029">0004-AMND-029</a>)
```
Comment (ze97286) [inline protocol/0004-AMND-amends.md]: it won't insert anything really, it will just expire it.

**Review by jeremyletang — DISMISSED** [review]

File: protocol/0004-AMND-amends.md [hunk 2]
```diff
@@ -22,6 +22,7 @@
 - Attempting to alter details on a filled order will cause the amend to be rejected (<a name="0004-AMND-017" href="#0004-AMND-017">0004-AMND-017</a>)
 - Attempting to alter details on a cancelled order will cause the amend to be rejected (<a name="0004-AMND-018" href="#0004-AMND-018">0004-AMND-018</a>)
 - Attempting to alter details on an expired order will cause the amend to be rejected (<a name="0004-AMND-019" href="#0004-AMND-019">0004-AMND-019</a>)
+- Amending expiry time of an active GTT order to a past time whilst also simultaneously amending the price and size of the GTT order will cause the order to immediately expire with the order details updated to reflect the order details requiring amendment (<a name="0004-AMND-029" href="#0004-AMND-029">0004-AMND-029</a>)
```
Comment (ze97286) [inline protocol/0004-AMND-amends.md]: I don't think the size should be part of this AC but maybe doesn't matter.

File: protocol/0004-AMND-amends.md (hunk 1 above)
Comment (vega-paul) [reply protocol/0004-AMND-amends.md]: @ze97286 updated again

File: protocol/0004-AMND-amends.md (hunk 2 above)
Comment (vega-paul) [reply protocol/0004-AMND-amends.md]: Updated @ze97286

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**